ocurrent-plugin/current_ocluster.ml

```diff
@@ -192,8 +192,8 @@ let build ?cache_hint t ~pool ~src ~options dockerfile =
   and> src = src in
   Raw.build ?cache_hint t ~pool ~src ~options dockerfile
 
-let build_obuilder ?cache_hint t ~pool ~src spec =
-  Current.component "obuild@,%s" pool |>
+let build_obuilder ?(label="") ?cache_hint t ~pool ~src spec =
+  Current.component "obuild@,%s@,%s" pool label |>
```

Comment:
I think this will output an extra newline when the label is empty. Maybe something like `Fmt.(option (cut ++ string))` would help.

Reply:
Yes indeed. I updated the PR and also added the same kind of change for `build` and `build_and_push` for completeness.